### PR TITLE
Added checking for incompatable libraries

### PR DESCRIPTION
**Pull request by rf20008**
## Summary

This pull request is for adding checking for incompatible libraries in client.run()  which raises Warnings if incompatible libraries are added

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [x] This PR is a breaking change (e.g. methods or parameters removed/renamed) (potentially)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by TAG-Epic** [comment]
Hey! Please test your changes.

File: nextcord/client.py [hunk 1]
```diff
@@ -632,6 +636,13 @@ def run(self, *args: Any, **kwargs: Any) -> None:
             is blocking. That means that registration of events or anything being
             called after this function call will not execute until it returns.
         """
+        
+        for libraryName in ["discord", "discord_slash", "pyfork", "enhanced-dpy"]: #more imcompatable libraries to be added
```
Comment (TAG-Epic) [inline nextcord/client.py:640]: This should be snake case

Comment (TAG-Epic) [reply nextcord/client.py:640]: pyfork and discord.py should be added

Comment (TAG-Epic) [reply nextcord/client.py:640]: The comment should be removed

**Comment by ooliver1** [comment]
why did you close this btw?

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

File: nextcord/client.py (hunk 1 above)
Comment (rf20008) [reply nextcord/client.py:640]: > why did you close this btw?

renamed branch due to spelling error in branch name

**Comment by rf20008** [comment]
I made the changes, however this loop turns the client object into a string for some reason

File: nextcord/client.py [hunk 2]
```diff
@@ -632,6 +636,13 @@ def run(self, *args: Any, **kwargs: Any) -> None:
...
+            try:
+                dist = pkg_resources.get_distrubtion("discord")
```
Comment (ooliver1) [inline nextcord/client.py:642]: > `get_distrubtion`
should be get_distribtion iirc

**Pull request closed without merging**